script_library/deployment-actions-kvm.sh

```diff
@@ -56,7 +56,8 @@ function deploy_airship(){
 }
 function add_compute(){
     echo "Now Add compute"
-    run_ansible ${socok8s_absolute_dir}/playbooks/add_compute.yml
+    tagged_info=" --tags add_computenode"
```

Comment:
for consistency in the consistency, can we use use "add_compute" or "add_compute_node" so we have underscore between words in the names?